providers/mongodb.js

```diff
@@ -85,3 +85,18 @@ module.exports = class extends Provider {
 };
 
 const resolveQuery = query => isObject(query) ? query : { id: query };
+
+const parseUpdateObject = (doc, pref = '', oldObj = {}) => {
+	const obj = oldObj;
+	const prefix = pref !== '' ? `${pref}.` : '';
+	for (const key in doc) {
+		if (Object.prototype.hasOwnProperty.call(doc, key)) {
+			if (typeof doc[key] !== 'object' || Object.keys(doc[key]).length === 0) {
+				obj[`${prefix}${key}`] = doc[key];
+				continue;
```

Comment:
this is useless

Reply:
the hasOwnProperty call? I was just making eslint happy :3 If you want it to get removed, I'll do it.

Reply:
No. reviews are always the last line.
The `continue`